js/datatable.js

```diff
@@ -1,38 +1,80 @@
 
-  ["dog", "Irish WolfHound", "img/dog-1.jpg"],
-  ["dog", "Crested Hairless", "img/dog-2.jpg"],
-  ["dog", "Golden Retriever", "img/dog-3.jpg"],
+  ["dog", "#B2B2B2",
+    ["Irish WolfHound", "img/dog-1.jpg"],
+    ["Crested Hairless", "img/dog-2.jpg"],
+    ["Golden Retriever", "img/dog-3.jpg"]],
 
-  ["germ", "Common Cold", "img/germ-1.jpg"],
-  ["germ", "Germs", "img/germ-2.jpg"],
-  ["germ", "Germs", "img/germ-3.jpg"],
+  ["germ", "#006600",
+    ["Common Cold", "img/germ-1.jpg"],
+    ["Germs", "img/germ-2.jpg"],
+    ["Germs", "img/germ-3.jpg"]],
 
-  ["heights", "heights", "img/heights-1.jpg"],
-  ["heights", "heights", "img/heights-2.jpg"],
-  ["heights", "heights", "img/heights-3.jpg"],
+  ["heights", "#6699FF",
+    ["heights", "img/heights-1.jpg"],
+    ["heights", "img/heights-2.jpg"],
+    ["heights", "img/heights-3.jpg"]],
 
-  ["kitten", "Kittens", "img/kitten-1.jpg"],
-  ["kitten", "Kittens", "img/kitten-2.jpg"],
+  ["kitten", "#FF3399",
+    ["Kittens", "img/kitten-1.jpg"],
+    ["Kittens", "img/kitten-2.jpg"]],
 
-  ["needle", "Needle", "img/needles-1.jpg"],
-  ["needle", "Needle", "img/needles-2.jpg"],
-  ["needle", "Needle", "img/needles-3.jpg"],
+  ["needle", "#FF3300",
+    ["Needle", "img/needles-1.jpg"],
+    ["Needle", "img/needles-2.jpg"],
+    ["Needle", "img/needles-3.jpg"]],
 
-  ["snake", "Snake", "img/snake-1.jpg"],
-  ["snake", "Snake", "img/snake-2.jpg"],
+  ["snake", "#669900",
+    ["Snake", "img/snake-1.jpg"],
+    ["Snake", "img/snake-2.jpg"]],
 
-  ["spider", "Spider", "img/spider-1.jpg"],
-  ["spider", "Spider", "img/spider-2.jpg"],
-  ["spider", "Spider", "img/spider-3.jpg"],
+  ["spider", "#855C33",
+    ["Spider", "img/spider-1.jpg"],
+    ["Spider", "img/spider-2.jpg"],
+    ["Spider", "img/spider-3.jpg"]]
 ];
 
+
+// var dataTable = [
+//   ["clown", "Pennywise", "img/clown-1.jpg"],
+//   ["clown", "Ronald Mcdonald", "img/clown-2.jpg"],
+//   ["clown", "Circus Clowns", "img/clown-3.jpg"],
+//   ["clown", "Twisty", "img/clown-4.jpg"],
+//
+//   ["dog", "Irish WolfHound", "img/dog-1.jpg"],
+//   ["dog", "Crested Hairless", "img/dog-2.jpg"],
+//   ["dog", "Golden Retriever", "img/dog-3.jpg"],
+//
+//   ["germ", "Common Cold", "img/germ-1.jpg"],
+//   ["germ", "Germs", "img/germ-2.jpg"],
+//   ["germ", "Germs", "img/germ-3.jpg"],
+//
+  // ["heights", "heights", "img/heights-1.jpg"],
+  // ["heights", "heights", "img/heights-2.jpg"],
+  // ["heights", "heights", "img/heights-3.jpg"],
+  //
+  // ["kitten", "Kittens", "img/kitten-1.jpg"],
+  // ["kitten", "Kittens", "img/kitten-2.jpg"],
+  //
+  // ["needle", "Needle", "img/needles-1.jpg"],
+  // ["needle", "Needle", "img/needles-2.jpg"],
+  // ["needle", "Needle", "img/needles-3.jpg"],
+  //
+  // ["snake", "Snake", "img/snake-1.jpg"],
+  // ["snake", "Snake", "img/snake-2.jpg"],
+  //
+  // ["spider", "Spider", "img/spider-1.jpg"],
+  // ["spider", "Spider", "img/spider-2.jpg"],
+  // ["spider", "Spider", "img/spider-3.jpg"],
+// ];
+
```

Comment:
Avoid leaving large chunks of commented out code in your master branch. It decreases readability for future users of your code. It's fine to remove it in a descriptive commit, and you can then always return to it later if you need it. `removed old dataTable` - easy to see and search in the commit history.